packages/twenty-front/src/modules/workflow/types/Workflow.ts

```diff
@@ -69,6 +69,7 @@ export type WorkflowManualTrigger = BaseTrigger & {
   type: 'MANUAL';
   settings: {
     objectType?: string;
+    outputSchema?: object;
```

Comment:
why objectType and outputSchema are optional?

Reply:
Ok then can you do that also for other step and triggers and initialize them to null ? Double check that the resolver endpoint allows null values also

Reply:
did the opposite actually because the endpoint cannot return null